reference/widgets.v3.yml

```diff
@@ -502,7 +510,7 @@ components:
         type: string
     FilterTemplateFileParam:
       in: query
-      name: template_file
+      name: template_file 
```

Comment:
From the ticket, it seems like this query parameter should also be added to the [Get Theme Regions](https://developer.bigcommerce.com/docs/rest-content/widgets/regions) endpoint? Currently, the docs have `templateFile` as the only query parameter